inyoka_theme_default/templates/portal/usercp/subscriptions.html

```diff
@@ -0,0 +1,110 @@
+{#
+    portal/usercp/subscriptions.html
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    This page shows all subscriptions of the user and let him
+    manage them.
+
+    :copyright: (c) 2007-2015 by the Inyoka Team, see AUTHORS for more details.
+    :license: BSD, see LICENSE for more details.
+#}
+
+{% extends 'portal/usercp/overall.html' %}
+{% import 'macros.html' as macros %}
+{% set rendered_pagination = macros.render_pagination(pagination) %}
```

Comment:
already done in l.34. Removing this one would also save the import of 'macros.html'